ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerTest.scala

```diff
@@ -114,7 +136,7 @@ object BigSamplerTest extends Properties("BigSampler") {
     "double_field")
 
   property("should hash on supported required types in Avro") = forAll(richAvroGen) { r =>
-    val hasher = newTestHasher()
+    val hasher = newTestFarmHasher()
```

Comment:
Not necessarily in the scope of this PR but would be nice to either map across both hashers or gen them instead.

Reply:
I"m not sure it's in scope either but it sounds fun and relatively quick to use Gen.oneOf so I'll see what I can do